### DISCUSSION TITLE
OpenAPI Country restrictions?

### DISCUSSION THREAD

**Discussion by madice6** [OAI/OpenAPI-Specification#4118] [Q&A]
Hi! I am currently in the Philippines and I'm experiencing this error message:

{"code":-1002,"msg":"You are not authorized to execute this request"}

I'm thinking it has something to do with the VPN I use.

Thanks. Can anyone help?

**Comment by handrews** [comment]
@madice6 this repository is for the OpenAPI specification itself- I'm afraid we cannot help with any specific tool or API.  Please contact your tool or API provider for assistance.  Or possibly network administrator if it's a VPN issue.
